ntoskrnl/fsrtl/fastio.c

```diff
@@ -1605,7 +1605,11 @@ FsRtlAcquireFileExclusiveCommon(IN PFILE_OBJECT FileObject,
             FilterCallbacks->PostAcquireForSectionSynchronization(&CbData, Status, CompletionContext);
         }
 
-        return Status;
+        /* Return here only in specific status cases */
```

Comment:
I would tend to think that this comment could explain a bit more the "why" , instead of the "what".

Reply:
OK, can I shortly tell here "since this is expected to be returned from FltMgr"? Or need I explain a bit more what exactly happens inside FltMgr and hence why this is returned? I'm not even sure where this is returned from FltMgr yet, it needs more investigations.

Reply:
For the reference: this status is returned from Filter Callbacks those are initialized and executed inside FltMgr.

Reply:
OK, improved the comment. Now it tells a bit about the statuses checked. Found the explanation in our FastFat driver (which also has these callbacks implemented, same as cdfs and ext2fs drivers): https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/filesystems/fastfat/resrcsup.c;hb=64b58397f3694d3234db5de313854e67be69728f#l946. Toke some wording from there.